Sources/INNavigation/Navigation/Helper/View+VerticalNavigation.swift

```diff
@@ -23,15 +23,20 @@ extension View {
 		.sheet(
 			item: Binding<PresentationInfo?>(
 				get: {
-					guard binding.wrappedValue?.presentationType == .sheet else {
+					guard binding.wrappedValue?.presentationType != .fullScreen else {
 						return nil
 					}
 					return binding.wrappedValue
 				}, set: { _ in }
 			),
 			onDismiss: binding.wrappedValue?.onDismiss,
 			content: { presentationItem in
-				RouterView(index: presentationItem.index)
+				if case let .sheet(detent) = presentationItem.presentationType {
```

Comment:
It's a set, therefore, plural
```suggestion
				if case let .sheet(detents) = presentationItem.presentationType {
```